tools/export/iar.py

```diff
@@ -46,7 +46,7 @@ class IAREmbeddedWorkbench(Exporter):
             # target is not supported yet
             continue
 
-    def generate(self):
+    def generate(self, **kwargs):
```

Comment:
Oversight from initial round of hooking this is in, should definitely be a named arg. Will change in next commit.